test/sol6/kyberNetwork.js

```diff
@@ -2947,6 +2949,40 @@ contract('KyberNetwork', function(accounts) {
                 "rate < min rate"
             );
         });
+
+        it("trade dest qty == 0", async function() {
```

Comment:
please use title

check that when one reserve has zero qty. getExpected rate returns 0. trade reverts

Reply:
Updated